applications/portfolioDrafts/createPortfolioDraft.ts

```diff
@@ -29,6 +29,7 @@ export const handler = async (event: APIGatewayProxyEvent): Promise<APIGatewayPr
     created_at: now,
     updated_at: now,
     status: ProvisioningStatus.NotStarted,
+    num_portfolio_managers: 0,
```

Comment:
Something to consider for future refactoring: having a `default()` or `empty()` or something factory for these items.

Reply:
Actually, this feels like a computed/derviced (not stored) property

Reply:
Yes, Jeff had suggested that perhaps another function might perform this computation.  When we have users the creating function should set it to one and add the creator as the default manager.

Reply:
And that might be overkill as a separate function, but perhaps rolled into function that updates the managers.